infrastructure/cloudtrail_bucket_policy.tpl

```diff
@@ -11,18 +11,34 @@
       "Resource": ["arn:aws:s3:::${bucketName}"]
     },
     {
-      "Sid": "AWSCloudTrailWrite20150319",
+      "Sid": "Grant CloudTrail Permission to Write Logs",
```

Comment:
the documentation says "the Sid element supports ASCII uppercase letters (A-Z), lowercase letters (a-z), and numbers (0-9)." So you might need to change these

Reply:
It seems spaces are allowed in Sids for some policies and not others (e.g. KMS policies allow spaces and IAM policies do not.) I will remove the spaces for consistency across the codebase.